doc/source/conf.py

```diff
@@ -41,6 +41,11 @@
 import sunpy.data
 sunpy.data.download_sample_data(overwrite=False)
 
+# -- Shut up numpy warnings from WCSAxes
+
+import numpy as np
+np.seterr(invalid='ignore')
```

Comment:
Thinking about this a bit more, I remember that it may already been fixed in astropy master. If it was python3.6 and wcsaxes related.